### DISCUSSION TITLE
AttributeError: partially initialized module 'flet' has no attribute 'Page' (most likely due to a circular import)[Question]

### DISCUSSION THREAD

**Discussion by xBreadyx** [flet-dev/flet#1634] [Q&A] [answered]
### Question

Cannot run the test case
<img width="633" alt="Screenshot_2" src="https://github.com/flet-dev/flet/assets/19890479/cf63af5d-fb4f-4c53-8789-2a37dbf84262">

### Code sample
import flet as ft

def main(page: ft.Page):
    # add/update controls on Page
    pass

ft.app(target=main)
_No response_

### Error message
AttributeError: partially initialized module 'flet' has no attribute 'Page' (most likely due to a circular import)
_No response_

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Make sure the python file you are running is not name is not `flet.py`!
